File: PRAK402-2310817210019-NaufalElyzar.c
#include <stdio.h>

int genap(int angka){
    for(int i = angka; i >= 1; --i) {
        if (i % 2 == 0) {
            printf("%d ", i);
        }
    }
    printf("\n");
}

int ganjil(int angka){
    for(int i = 1; i <= angka; ++i) {
        if (i % 2 != 0) {
            printf("%d ", i);
        }
    }
    printf("\n");
}

int main(){
    int angka;
    scanf("%d", &angka);
    ganjil(angka);
    genap(angka);
}

File: PRAK402-2310817210019-NaufalElyzar.py
def genap(angka):
    for i in range(angka, 0, -1):
        if i % 2 == 0:
            print(i, end=" ")
    print()

def ganjil(angka):
    for i in range(1, angka + 1):
        if i % 2 != 0:
            print(i, end=" ")
    print()

def main():
    angka = int(input())
    ganjil(angka)
    genap(angka)

if __name__ == "__main__":
    main()
